### PR TITLE
[daint-gpu] Gromacs

### PR DESCRIPTION
**Pull request by victorusu**
There were some old GROMACS recipes in this branch. I didn't know.

Should I keep or remove them? or should I close this PR and use another branch name?

@sebkelle1, you are the one who created the old recipes? What do I do?

### CODE REVIEW

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2018.8-CrayGNU-19.10-cuda-10.1.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+local_common_options = '-DCMAKE_BUILD_TYPE=Release -DGPU_DEPLOYMENT_KIT_ROOT_DIR=$CUDATOOLKIT_HOME -DGMX_BUILD_OWN_FFTW=ON -DGMX_OPENMP=ON -DGMX_GPU=ON -DGMX_PREFER_STATIC_LIBS=ON -DGMX_SIMD=AVX2_256'
+configopts = ['%s -DGMX_MPI=OFF ' % local_common_options,
+              '%s -DGMX_MPI=ON ' % local_common_options]
+
+skipsteps = ['test']
+
+# CMake dependency with dummy toolchain
+builddependencies = [
+    ('CMake', '3.14.1', '', True),
```
Comment (lucamar) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2018.8-CrayGNU-19.10-cuda-10.1.eb]:
```suggestion
    ('CMake', '3.14.5', '', True),
```

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2018.8-CrayGNU-19.10.eb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+sources = [SOURCELOWER_TAR_GZ]
+
+local_common_options = '-DCMAKE_BUILD_TYPE=Release -DGMX_BUILD_OWN_FFTW=ON -DGMX_OPENMP=ON -DGMX_GPU=OFF -DGMX_PREFER_STATIC_LIBS=ON -DGMX_SIMD=AVX2_256'
+configopts = ['%s -DGMX_MPI=OFF ' % local_common_options,
+              '%s -DGMX_MPI=ON ' % local_common_options]
+
+skipsteps = ['test']
+
+builddependencies = [
+    ('CMake', '3.14.1', '', True),
```
Comment (lucamar) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2018.8-CrayGNU-19.10.eb]:
```suggestion
    ('CMake', '3.14.5', '', True),
```

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2019.1-CrayGNU-19.03-cuda-10.0-scorep.eb [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+configopts += "-DCUDA_HOST_COMPILER=$(which CC) "
+configopts += "-DBUILD_TESTING=OFF "
+# user instrumentation doesn't work atm
+#configopts += "-DCMAKE_CXX_FLAGS=\"${CXXFLAGS} --user\" "
+
+skipsteps = ['test']
+
+# CMake dependency with dummy toolchain
+builddependencies = [
+    ('CMake', '3.14.1', '', True),
```
Comment (lucamar) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2019.1-CrayGNU-19.03-cuda-10.0-scorep.eb]:
```suggestion
    ('CMake', '3.14.5', '', True),
```

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2019.1-CrayIntel-18.08.eb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+patches = ['gromacs-2018.2.patch']
+
+configopts = '-DGMX_CYCLE_SUBCOUNTERS=ON -DGMX_BUILD_OWN_FFTW=ON -DGMX_GPU=OFF -DGMX_MPI=OFF'
+
+skipsteps = ['test']
+
+# CMake dependency with dummy toolchain
+builddependencies = [
+    ('CMake', '3.13.3', '', True),
```
Comment (lucamar) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2019.1-CrayIntel-18.08.eb]:
```suggestion
    ('CMake', '3.14.5', '', True),
```

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2019.4-CrayGNU-19.10-cuda-10.1.eb [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+local_common_options = '-DCMAKE_BUILD_TYPE=Release -DGPU_DEPLOYMENT_KIT_ROOT_DIR=$CUDATOOLKIT_HOME -DGMX_BUILD_OWN_FFTW=ON -DGMX_OPENMP=ON -DGMX_GPU=ON -DGMX_PREFER_STATIC_LIBS=ON -DGMX_SIMD=AVX2_256'
+configopts = ['%s -DGMX_MPI=OFF ' % local_common_options,
+              '%s -DGMX_MPI=ON ' % local_common_options]
+
+skipsteps = ['test']
+
+# CMake dependency with dummy toolchain
+builddependencies = [
+    ('CMake', '3.14.1', '', True),
```
Comment (lucamar) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2019.4-CrayGNU-19.10-cuda-10.1.eb]:
```suggestion
    ('CMake', '3.14.5', '', True),
```

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2019.4-CrayGNU-19.10.eb [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+local_common_options = '-DCMAKE_BUILD_TYPE=Release -DGMX_BUILD_OWN_FFTW=ON -DGMX_OPENMP=ON -DGMX_GPU=OFF -DGMX_PREFER_STATIC_LIBS=ON -DGMX_SIMD=AVX2_256'
+configopts = ['%s -DGMX_MPI=OFF ' % local_common_options,
+              '%s -DGMX_MPI=ON ' % local_common_options]
+
+skipsteps = ['test']
+
+# CMake dependency with dummy toolchain
+builddependencies = [
+    ('CMake', '3.14.1', '', True),
```
Comment (lucamar) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2019.4-CrayGNU-19.10.eb]:
```suggestion
    ('CMake', '3.14.5', '', True),
```

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2019.1-CrayGNU-19.03-cuda-10.0-scorep.eb [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Kenneth Hoste
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-93.html
+#
+name = 'GROMACS'
+version = "2019.1"
+cudaversion = '10.0'
```
Comment (lucamar) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2019.1-CrayGNU-19.03-cuda-10.0-scorep.eb]:
```suggestion
local_cudaversion = '10.0'
```

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2019.1-CrayGNU-19.03-cuda-10.0-scorep.eb [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+versionsuffix = '-cuda-%s-scorep' % cudaversion
```
Comment (lucamar) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2019.1-CrayGNU-19.03-cuda-10.0-scorep.eb]:
```suggestion
versionsuffix = '-cuda-%s-scorep' % local_cudaversion
```

**Comment by victorusu** [comment]
I am closing the PR since it is cluttered with multiple files that do not belong to this PR

**Pull request closed without merging**